newIDE/app/src/UI/Theme/index.js

```diff
@@ -1,23 +1,61 @@
 // @flow
+import {
+  select,
+  blend as Install,
+} from '../../Utils/Object';
+import Language from '../../Utils/i18n/language';
+import { stash } from './domain';
+import DefaultTheme, {
+  themeName as $default,
```

Comment:
Let's avoid naming with a $, that implies some kind of special value à la jQuery. Instead, let's name if "defaultThemeName" - it's more descriptive.